docs/book_files/csi_block_storage_kc_whatsnew.md

```diff
@@ -1,8 +1,8 @@
 # What's new
 
-This topic lists the dates and nature of updates to the published information of IBM® block storage CSI driver 1.9.0.
+This topic lists the dates and nature of updates to the published information of IBM® block storage CSI driver 1.10.0.
 
 |Date|Nature of updates to the published information|
 |----|----------------------------------------------|
-|17 March 2022|The version information was added to the IBM Documentation.<br>For more information about this version, see the [What's new in 1.9.0](../content/release_notes/whats_new.md) section in the release notes.|
+|15 July 2022|The version information was added to the IBM Documentation.<br>For more information about this version, see the [What's new in 1.10.0](../content/release_notes/whats_new.md) section in the release notes.|
```

Comment:
AFAIK the 26th has always been the official date

Reply:
done

Reply:
I still [see 15](https://github.com/IBM/ibm-block-csi-driver/blob/2dc6c84cc87354161d0107f8fdeb57bbb7c2f741/docs/content/release_notes/whats_new.md#whats-new-in-1100)

Reply:
I don't know why but in any case we can resolve this issue. it will be updated again before final release.

Reply:
this PR is for merge to `develop`. it should not contain "15 July" in `develop`.
we can leave this open until it is updated again

Reply:
let me know if you need any help pushing the update

Reply:
this is great, but due to GitHub limitations, technically (and perhaps by design), one cannot open a single conversation for multiple places.
still, the need for such conversations is common, so that a review could be more high-level, when needed.
this is why [I linked](https://github.com/IBM/ibm-block-csi-driver/pull/532#discussion_r918943685) to a different file in the same conversation.
please always use Ctrl+F under the `Files changed` tab [before writing "done"](https://github.com/IBM/ibm-block-csi-driver/pull/517#discussion_r911953786)

Reply:
I didn't realize it was to a different section. I don't write the date in the other section for most of my other products.
Next time please write in the name of the topic (what's new, or RN what's new) as the link text so it's clearer. thanks!
it's updated now